rpmUtils/updates.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, instlist, availlist):
         
         self._archlist = rpmUtils.arch.getArchList(self.myarch)
 
-        self._multilib_compat_arches = rpmUtils.arch.getMultiArchInfo(self.myarch)
+        self._multilib_compat_archs = rpmUtils.arch.getMultiArchInfo(self.myarch)
 
```

Comment:
Again, even though this is "private" to even think about changing this we'd need to be pretty sure that nobody is using it.